### PR TITLE
feat: Add Column innerText and value font props

### LINKED ISSUE #3568: Request feature for bar chart
We would like to request for the bar chart to have the following features:

- Allow label to be placed in the bar chart along with the value on the right most part. If the label is longer than the bar chart it should be ellipsized. The value has more priority than the label.

- To be able to add an icon on the left side of the y-axis.

Here's the proposed image:
image

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes #3568

Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [x] I have raised new issues to address them separately

### CODE REVIEW

**Comment by jbrundage** [comment]
image

**Comment by jbrundage** [comment]
@GordonSmith this is passing and ready for review

File: demos/gallery/samples/chart/InnerTextBar.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Bar } from "@hpcc-js/chart";
+
+new Bar()
+    .innerText({
+        "a": ["some text","lorem ipsum"],
+        "b": ["more text","dolor sit amet"],
+        "c": ["other text","consectetur adipisicing elit"]
+    })
```
Comment (GordonSmith) [inline demos/gallery/samples/chart/InnerTextBar.js:8]: @jbrundage `innerText` should not needed as the current axis text should just work (we don't have inner text for Pie for example).

File: demos/gallery/samples/chart/InnerTextBar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    .target("target")
+    .columns(["A","B","C"])
+    .data([
+        ["a",2,-3],
+        ["b",5,-6],
+        ["c",8,9],
+    ])
```
Comment (GordonSmith) [inline demos/gallery/samples/chart/InnerTextBar.js:15]: So the data would look like:
```
    .data([
        ["lorem ipsum",2,-3],
        ["dolor sit amet",5,-6],
        ["consectetur adipisicing elit",8,9],
    ])
```

**Pull request merged**